package.json

```diff
@@ -79,6 +79,10 @@
         "metals.scalafmtConfigPath": {
           "type": "string",
           "markdownDescription": "Optional custom path to the .scalafmt.conf file.\n\nShould be relative to the workspace root directory and use forward slashes `/` for file separators (even on Windows)."
+        },
+        "metals.customRepositories": {
+          "type": "string",
+          "markdownDescription": "Optional space separated list of custom resolvers passed to coursier when fetching metals dependencies."
```

Comment:
It's not clear from this description what syntax is valid for custom repositories. What do you think about adding an example or a link to the Coursier documentation (https://get-coursier.io/docs/other-repositories)?